### DISCUSSION TITLE
上海一名小粉红想去美领馆涂鸦抗议佩洛西访台，结果被公安带走喝茶了。你也配爱国？🤣

### DISCUSSION THREAD

**Discussion by ghost** [WeNeedHome/SummaryOfLoanSuspension#1083] [General]
告诉我，你爱什么国？
![image](https://user-images.githubusercontent.com/110222065/182546544-4891c98d-221f-4739-8135-4984c877b4e2.png)
![image](https://user-images.githubusercontent.com/110222065/182546562-a3625c40-be45-4b31-ab37-d9677c04136b.png)

**Comment by canghaicode** [comment]
我感恩还不行吗

**Reply by hclgdb** [reply]
Win!Win!Win!

**Comment by ChinaChenMingQuan** [comment]
去去去，给我干活去，大人的事什么时候轮到你个小屁孩来管了！今早要是我少了一块钱不能给车加油，今晚我回来保证打烂你的屁屁！【狗头】

**Comment by wedxz** [comment]
让那些小粉红继续冲啊

**Comment by ChrisJonasou** [comment]
乐

**Comment by volans19** [comment]
他还真以为中国能抗议，90年的，这么大了还这么脑瘫

**Comment by 505Games-oss** [comment]
我估摸着2012年的U型锁事件肯定会再度发生，这次被砸的会是谁呢，让我们拭目以待

**Comment by JanusTida** [comment]
就事论事地说，他去公共场合乱涂乱画，无论是表达什么都是不对的，抓起来教育一下没问题。

**Comment by xiejun946** [comment]
算了，这次中国应对做的够可以了，没有怂

**Comment by clleo888** [comment]
小粉红不配爱国, 但不爱国又有罪, 做不了人了...

**Comment by yjg5035** [comment]
还以为怎么处理了呢，结果就这样？不管怎么骂，中国还是法制社会，这种行为肯定要被带局子里去的，不然舆论上说不过去。进去了，走个流程就放了，正常操作。

**Reply by clleo888** [reply]
"进去了，走个流程就放了，正常操作" ---> 这叫法制社会, 服了....
